src/index.html

```diff
@@ -17,16 +17,32 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action="#"
+      method="get"
+      class="searchBar searchBar--big"
```

Comment:
The `class` attribute has the value 'searchBar searchBar--big'. A better naming convention would be 'search-bar search-bar--big' to improve readability.